hepcrawl/utils.py

```diff
@@ -144,6 +144,28 @@ def get_first(iterable, default=None):
     return default
 
 
+def best_match(iterable, default=None):
+    """Get first truthy value from iterable, fall back to default.
```

Comment:
The name of the function looks a bit misleading, maybe it's better to call it something like `get_first` or similar, wdyt?

Reply:
Somebody already did something similar xd
    https://pypi.python.org/pypi/first

(not that we sholud use this, it's simple enough here, I don't think it's worth it to add an extra dependency).

Reply:
I took the name from Flask, but :+1: to change it as it is more generic.